app/test/e2e/dataset-clone.spec.js

```diff
@@ -24,12 +24,16 @@ describe('Dataset clone tests', () => {
     });
 
     it('Clone a dataset that does not exist should return a 404 error', async () => {
+        nock(process.env.CT_URL, { reqheaders: { authorization: 'Bearer abcd' } })
```

Comment:
I don't like that we have to call these 3 lines at the beginning of each test - if for some reason we have to make a change to the way we are mocking this, we are going to have to do a massive refactor to all calls of this in all microservices.

I'd suggest extracting this logic to a helper method in the `tests/utils` directory. Or, (might be a good but controversial idea :thinking:), what about exporting a function from the `'rw-api-microservice-node'` package? After all, that is the place where this logic lives right now... Somthing to think about :)

Reply:
Also, keep in mind that for test suites where all tests require this call, you can just do it in the `beforeEach` method of the test suite (I know this is not the case but for the future):
```javascript
const { mockTokenCall } = require('rw-api-microservice-node'); // require('./utils');
(...)
beforeEach(mockTokenCall);
```

Reply:
having this at the start of every test: necessary evil, as not all mocks are alike - not all of them return the same profile

moving this into a helper function: done

moving this into the lib: too opinionated, as now you're forcing the usage of nock

beforeEach: DRY, but I still prefer less DRY but more readable code.